rspack.config.ts

```diff
@@ -271,37 +240,23 @@ const appConfig: Configuration = {
       },
       {
         test: /\.pegjs/,
-        use: {loader: 'pegjs-loader'},
+        loader: 'pegjs-loader',
       },
       {
         test: /\.css/,
-        use: ['style-loader', 'css-loader'],
+        type: 'css',
       },
```

Comment:
I'm curious why sentry use style-loader to handle css and use minicssExtractPlugin to handle less before?